components/bitcoin-topics/topic/TopicHeader.tsx

```diff
@@ -1,16 +1,21 @@
 "use client"
 
-import Metadata from "./metadata"
+import { usePathname } from "next/navigation"
 import Summary from "./summary"
 
 export function TopicHeader({ title, tags, summary }: any) {
+    const pathname = usePathname()
+    const isBitcoinHistory = pathname === "/topics/bitcoin-history"
```

Comment:
There should be a better way for a topic to opt out of this rendering without having it explicitly defined here.
A way to do that could be to have a `renderTitle` field in the frontmatter of the topic.
But that means a change to all the topics markdowns and for such a small nit I don't think its advisable.

We can think about things like this going forward.

Reply:
You're right, this isn't ideal
for now, it's not worth changing all markdown files for this ig